test/test_buffer_protocol.py

```diff
@@ -0,0 +1,175 @@
+import torch.testing._internal.common_utils as common
+from torch.testing._internal.common_device_type import (
+    instantiate_device_type_tests,
+    dtypes, onlyCPU
+)
+
+import torch
+import numpy
+
+def get_dtype_size(dtype):
+    return int(torch.empty((), dtype=dtype).element_size())
```

Comment:
Not for this PR, but I wonder if we should let users acquire a dtypes info directly from it. So, like, `torch.float32.info` would be equivalent to `torch.finfo(torch.float32)`.

Reply:
I feel like that would probably be a good thing. Not sure about how frequently it shows up on users' code, though. By the way, do complex values have anything similar: `torch.cinfo` (didn't find it)?

Reply:
A natural question. The complex types are accessed using finfo, actually. This is consistent with NumPy.

Reply:
Ah, I guess I missed the fact that it was possible to pass an argument to `torch.finfo` constructor.